designer/client/components/Visualisation/visualisation.scss

```diff
@@ -3,6 +3,7 @@
 .visualisation {
   position: relative;
   flex: 1;
+  top: 110px;
```

Comment:
this needs to be padding-top. By using top + relative positioning, all that's happening is it's shifting the visualisation down by 110px. So 110px is going to be hidden at the bottom of the screen

Reply:
This is now padding top as requested